components/common/Table.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from "react";
+
+interface Student {
+  id: number;
+  firstName: string;
+  lastName: string;
+  email: string;
+  resumeLink: string;
+  detailsLink: string;
+}
+
+type PageProps = {
+  readonly Students: Student[];
+};
+
+const Table: FC<PageProps> = ({ Students }) => {
+  return (
+    <div className="max-w-2xl mx-auto">
```

Comment:
We can remove this div/containment as this component will be used in another container